docs/sources/datasources/_index.md

```diff
@@ -8,12 +8,14 @@ weight: 60
 
 # Data sources
 
-Grafana can query and integrate with many different types of databases. This is done by adding a **data source** of the type you want to query or integrate with.
-When you have created and configured a data source you are ready to start exploring and visualizing data, either in Explore or in a new Dashboard. A dashboard is composed of [panels]({{< relref "../panels-visualizations/" >}}), each panel contains a set of queries to one or more data sources.
+Grafana has built-in support for a wide range of _data sources_.
+These data sources might be files, services, or databases.
+You can also extend the supported data sources through one of the many [Plugins](/grafana/plugins/) (or build your own).
```

Comment:
wrong link?
I guess you wanted to link to https://grafana.com/grafana/plugins/

Reply:
"build your own" could be linked to https://grafana.com/docs/grafana/latest/developers/plugins/

Reply:
Good idea about adding the link to the build instructions. https://github.com/grafana/grafana/pull/61639/commits/b3e48d45922cf3331c71ef0c8fe6251901d59391